pkg/latte/ratelimit.go

```diff
@@ -0,0 +1,68 @@
+package latte
```

Comment:
Ratelimit may be changed recently. Use interaction directly to avoid stale code.

Reply:
Do we intend to move ratelimit to `latte` eventually? If so I could alias the functions to `interaction` package.

Reply:
We intent to use the same set of ratelimit eventually. For the time being, aliasing is a good idea.